frontend/controller/backend.js

```diff
@@ -125,7 +125,7 @@ sbp('sbp/selectors/register', {
         // if this isn't OP_CONTRACT, get latestHash, recreate and resend message
         if (!entry.isFirstMessage()) {
           const previousHEAD = await sbp('backend/latestHash', contractID)
-          entry = GIMessage.create(contractID, previousHEAD, entry.op())
+          entry = GIMessage.createV1_0(contractID, previousHEAD, entry.op())
```

Comment:
Will double-check this in my PR that I'm working on... we can merge this even if it's wrong though since it's a rare situation and I plan to fix it shortly.

If I verify the fix before your changes to my review are pushed, I'll try to remember to comment here with the real fix if it diverges from yours. This comment is mostly a reminder to myself.

Reply:
This actually is the correct fix. Great job @snowteamer! I made the same change in #1047.